introduction_to_amazon_algorithms/jumpstart-foundation-models/llama3.1-fine-tuning.ipynb

```diff
@@ -0,0 +1,707 @@
+{
```

Comment:
<p>Call out 405B Base and Instruct</p><p> </p><h1>Fine-tune Llama3.1 405B Base and Instruct Variants on SageMaker JumpStart</h1><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/4729/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>